CI/appveyor.install.ps1

```diff
@@ -223,10 +223,10 @@ function InstallLua() {
 }
 
 function InstallPcre() {
-  DownloadFile "https://sourceforge.net/projects/pcre/files/pcre/8.38/pcre-8.38.tar.gz/download" "pcre-8.38.tar.gz"
+  DownloadFile "https://ftp.pcre.org/pub/pcre/pcre-8.38.tar.gz" "pcre-8.38.tar.gz"
   ExtractTar "pcre-8.38.tar.gz" "pcre-8.38"
   Set-Location pcre-8.38\pcre-8.38
-  RunConfigure
+  RunConfigure "--enable-utf --enable-unicode-properties --prefix=$Env:MINGW_BASE_DIR_BASH"
```

Comment:
I am making an ass out of 'U' and "me" as it is my belief that the other CI platforms pull in packages (not compile from source, note!) that have those set as a reasonable set of features.

It is possible, with a couple of uses of, I think, `pcre_options(`\<option name>`, &`<integer set to 1 if option present>`)` to test for them - though I was in two minds about testing this - it would mean checking on start-up and putting up a `QMessageBox` saying that "this Mudlet executable is broken"...!

Reply:
Nah let's not worry about it for now.